### PR TITLE
Fix redo undo shortcut 

### LINKED ISSUE #234: add feedback for undo and redo shortcuts

### PR DESCRIPTION
**Pull request by Fyelight**
Closes #234 
by raising success message when no exception is thrown in  mainwindow

### CODE REVIEW

File: src/main/java/seedu/doit/ui/MainWindow.java [hunk 1]
```diff
@@ -97,24 +99,34 @@ private void setAccelerators() {
         this.scene.addEventFilter(KeyEvent.KEY_PRESSED, new EventHandler<KeyEvent>() {
             KeyCombination undo = new KeyCodeCombination(KeyCode.Z, KeyCombination.CONTROL_DOWN);
             KeyCombination redo = new KeyCodeCombination(KeyCode.Y, KeyCombination.CONTROL_DOWN);
+
             @Override
```
Comment (jinshunlee) [inline src/main/java/seedu/doit/ui/MainWindow.java]: maybe can change method name handleKeyEvent? also add some header comments here

**Review by jinshunlee — CHANGES_REQUESTED** [review]

**Comment by Fyelight** [comment]
@yexiexie  Add the comments since u did most of it.
I not really sure about the set accelerator really and why it is name that way.
Maybe you can tell us

**Comment by Fyelight** [comment]
@yexiexie  also add in your collate too.

**Comment by yexiexie** [comment]
method name setAccelerator() was there for help shortcut F1 linked to menu item. Now that we added some key events, method name can change to keyEvents to be more meaningful.

**Review by jinshunlee — APPROVED** [review]

**Pull request merged**